lambdas/subscribe/main.go

```diff
@@ -84,6 +85,14 @@ func Handler(ctx context.Context, request events.APIGatewayProxyRequest) (events
 		}, err
 	}
 
+	matched, err := regexp.MatchString(`^\+[1-9]\d{1,14}$`, requestBody.Number)
+	if err != nil || matched {
```

Comment:
Let's create a separate `if` block for `err != nil` that returns a `500` status code similar to how errors are handled throughout this function.

Reply:
@acthelemann 
OK, Separated.